shared/js/background/classes/site.es6.js

```diff
@@ -17,7 +17,7 @@ class Site {
     constructor (url) {
         this.url = url || ''
 
-        let domain = utils.extractHostFromURL(this.url) || ''
+        let domain = utils.extractHostFromURL(this.url, true) || ''
```

Comment:
Will this change have a side-effect on the UI? Is this domain field used in the popup?

Reply:
It does yes, perhaps that's the correct thing to do however. Given the user will be exempting the site for only the hostname rather than the entire domain.